prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-email-view/notification-email-view.component.html

```diff
@@ -11,14 +11,14 @@
     {{(editable) ? "Editing" : "Preview"}}
   </ng-container>
 
-  <button *ngIf="(Role.SUPER_ADMIN | inRole) && !editable"
+  <button *ngIf="((Role.SUPER_ADMIN | inRole) || (Role.MAINTENANCE | inRole)) && !editable"
```

Comment:
I would combine these buttons into single button.  Also, the `InRolePipe` can take singular or multiple roles: 

```html
  <button *ngIf="[Role.SUPER_ADMIN, Role.MAINTENANCE] | inRole"
          mat-icon-button
          [matTooltip]="(!editable) ? 'Edit Email Template' : 'Cancel Editing'"
          (click)="toggleEdit(!editable)">
    <mat-icon>{{ (!editable) ? 'edit' : 'close' }}</mat-icon>
  </button>
```

Reply:
Changed to toggle buttons and check for one role instead of 2.

By the way, this check `*ngIf="[Role.SUPER_ADMIN, Role.MAINTENANCE] | inRole` would not work if the user only has one of the roles, it evaluates to true if user has both `SUPER_ADMIN` and `MAINTENANCE` but evaluates to false if use has one or the other